src/main/java/teammates/common/datatransfer/attributes/StudentProfileAttributes.java

```diff
@@ -242,8 +244,8 @@ public StudentProfileAttributes build() {
             return profileAttributes;
         }
 
-        private boolean isGenderValid(String gender) {
-            return "male".equals(gender) || "female".equals(gender) || "other".equals(gender);
+        private boolean isGenderValid(gender gen) {
```

Comment:
This method isn't really necessary anymore, since an enum addresses the issue of validity.